modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -8,6 +8,7 @@ const jsDiff = require('diff');
 module.exports = {
   extend: '@apostrophecms/widget-type',
   options: {
+    icon: 'format-text-icon',
```

Comment:
I'm a little worried that we don't seem to have a safety net if they choose an icon here and it isn't registered. It would probably be done in AposAreaMenuItem. The dev should be catching this while building their site, but it'd be nice if we could catch it. Nothing actionable yet, maybe, but something to think about. CC @localghost443

Reply:
Yeah that's catchable. Would make a good followup ticket.